core/tests/bank_forks.rs

```diff
@@ -337,16 +337,17 @@ mod tests {
     #[test]
     fn test_slots_to_snapshot() {
         solana_logger::setup();
-        for add_root_interval in 1..10 {
+        let num_set_roots = MAX_CACHE_ENTRIES * 2;
+
+        for add_root_interval in &[1, 3, 9] {
```

Comment:
nits: preferably, the reason for the samples of `[1, 3, 9]` could be commented. Such a comment should explain: why not `[1, 4, 9]` nor `[1, 3, 19]`?